stuff2sort/System.py

```diff
@@ -0,0 +1,1150 @@
+                                                    )
+                                                    if match:
+                                                        a = int(match.group(1)) + int(
+                                                            offset
+                                                        )
+                                                        b = int(match.group(2)) + int(
+                                                            offset
+                                                        )
+                                                        if a > b:
+                                                            c = b
+                                                            b = a
+                                                            a = c
+                                                        found = False
+                                                        for element in connectivity_list:
+                                                            if int(element[0]) == a:
+                                                                if int(b) not in np.array(
+                                                                    element
+                                                                ).astype(int):
+                                                                    element.append(int(b))
+                                                                    found = True
+                                                        if not found:
+                                                            connectivity_list.append(
+                                                                [int(a), int(b)]
+                                                            )
+                                                    match = re.search(
+                                                        r"^\s*\n", line, flags=re.MULTILINE
+                                                    )
+                                                    if match:
+                                                        break
+                                            break
+                                break
+                    break
+        return connectivity_list
+
+    def write_highprec(self, gro_file):
+
+        '''
+        ------------------------------ \\
+        EFFECT: \\
+        --------------- \\
+        Function Writes A g96 File From The gro File \\
+        ------------------------------ \\
+        INPUT: \\
+        --------------- \\
+        gro_file: str -> Name Of The gro File \\
+        topology_file: str -> Name Of The Topology File \\
+        ------------------------------ \\
+        RETURN: \\
+        --------------- \\
+        filename: str -> Name Of The New g96 File \\
+        ------------------------------ \\
+        '''
+
+        # not tested yet, bc I didn't use a gro file AJ
```

Comment:
I like that you give your Initials in the comments, but sometimes you put # XX AJ <Comment> and other times you put AJ at the end. Just a remark that even in comments it would be nice if you keep to the same style.

Reply:
# AJ is for permanent comments, # XX AJ is for temporary comments, where one of us needs to do or decide something. That way we can search for XX to check if there's something left to discuss in the file.